concent_api/conductor/tasks.py

```diff
@@ -48,6 +51,12 @@ def blender_verification_request(
     # and result_package_path in the VerificationRequest have reports.
     verification_request.refresh_from_db()
 
+    subtask = Subtask.objects.get(
+        subtask_id = subtask_id,
+    )
+
```

Comment:
You have to handle exception which will raise if `get` result doesn't exist. Please check https://github.com/golemfactory/concent/pull/429/files#diff-d9a65f02de7b241b05a297d98193c439R20